code-submissions-clustering-plugin/src/main/kotlin/org/jetbrains/research/code/submissions/clustering/cli/LoadRunner.kt

```diff
@@ -0,0 +1,63 @@
+package org.jetbrains.research.code.submissions.clustering.cli
+
+import com.intellij.openapi.application.ApplicationStarter
+import com.intellij.openapi.diagnostic.Logger
+import com.xenomachina.argparser.ArgParser
+import org.jetbrains.kotlinx.dataframe.DataFrame
+import org.jetbrains.kotlinx.dataframe.api.column
+import org.jetbrains.kotlinx.dataframe.api.forEach
+import org.jetbrains.kotlinx.dataframe.api.getValue
+import org.jetbrains.kotlinx.dataframe.io.readCSV
+import org.jetbrains.research.code.submissions.clustering.Submission
+import org.jetbrains.research.code.submissions.clustering.buildGraph
+import org.jetbrains.research.code.submissions.clustering.load.Unifier
+import java.nio.file.Paths
+import kotlin.system.exitProcess
+
+object LoadRunner : ApplicationStarter {
+    private val logger = Logger.getInstance(this::class.java)
+    private lateinit var inputFile: String
+    private lateinit var outputDir: String
+
+    override fun getCommandName(): String = "load"
+
+    @Suppress("TooGenericExceptionCaught")
+    override fun main(args: MutableList<String>) {
+        try {
+            ArgParser(args.drop(1).toTypedArray()).parseInto(::TransformationsRunnerArgs).run {
+                inputFile = Paths.get(input).toString()
+                outputDir = Paths.get(output).toString().removeSuffix("/")
+            }
+
+            val df = DataFrame.readCSV(inputFile)
+            val id by column<Int>()
+            val step_id by column<Int>()
+            val code by column<String>()
+            val submissionsGraph = buildGraph(Unifier()) {
+                df.forEach {
+                    add(
+                        Submission(
+                            id = getValue(id),
+                            stepId = getValue(step_id),
+                            code = getValue(code)
+                        )
+                    )
+                }
+            }
+            submissionsGraph.print()
```

Comment:
Can we move into a function and make tests? we can compare the string representation for the graphs in the tests

For tests we should use special mock SDK for the python transformation. You can find examples [here](https://github.com/JetBrains-Research/Lupa/blob/main/python-analysers/src/test/kotlin/org/jetbrains/research/lupa/pythonAnalysis/callExpressions/analysis/psi/PyCallExpressionPsiAnalyzerTest.kt#L13)
ParametrizedBaseWithPythonSdkTest is from [this](https://github.com/JetBrains-Research/plugin-utilities/blob/main/plugin-utilities-test/src/main/kotlin/org/jetbrains/research/pluginUtilities/util/ParametrizedBaseWithPythonSdkTest.kt) repo